scout/server/blueprints/variant/templates/variant/fusion-variant.html

```diff
@@ -0,0 +1,464 @@
+{% extends "layout.html" %}
+{% from "utils.html" import comments_panel, activity_panel, pedigree_panel %}
+{% from "variant/variant_details.html" import old_observations %}
+{% from "variant/components.html" import external_scripts, external_stylesheets, matching_variants %}
+{% from "variant/utils.html" import modal_causative, overlapping_panel, genes_panel, transcripts_panel, sv_alignments, pin_button, causative_button, igv_track_selection %}
+{% from "variant/rank_score_results.html" import rankscore_panel %}
+{% from "variant/gene_disease_relations.html" import omim_phenotypes %}
+{% from "variant/variant_details.html" import gtcall_panel, frequencies, observations_panel %}
+{% from "variant/buttons.html" import variant_tag_button, dismiss_variant_button, splice_junctions_button %}
+
+{% block title %}
+  {{ super() }} - {{ institute.display_name }} - {{ case.display_name }} - {{ variant.display_name }}
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+  {{ external_stylesheets() }}
+{% endblock %}
+
+{% block top_nav %}
+  {{ super() }}
+  <li class="nav-item">
+    <a class="nav-link" href="{{ url_for('cases.index') }}">Institutes</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('overview.cases', institute_id=institute._id) }}">
+      {{ institute.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link text-nowrap" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}">
+      {{ case.display_name }}
+    </a>
+  </li>
+  <li class="nav-item">
+    {% if variant.category == "fusion" %}
```

Comment:
But if you are using this template you already know that variant.category == "fusion", no?